api/can_api_v2_definition.py

```diff
@@ -158,6 +158,10 @@ class RegionSummary(base_model.APIBaseModel):
     lat: Optional[float] = pydantic.Field(
         ..., description="Latitude of point within the state or county"
     )
+    locationId: str = pydantic.Field(
+        ...,
+        description="Location ID as defined here: https://github.com/covidatlas/li/blob/master/docs/reports-v1.md#general-notes",
```

Comment:
Took this description from the docs here: https://github.com/covid-projections/covid-data-public/blob/master/covidactnow/datapublic/common_fields.py#L39-L41